src/views/Ifos/components/IfoVesting/index.tsx

```diff
@@ -26,16 +27,13 @@ const VestingCardBody = styled(CardBody)`
   overflow-y: auto;
   max-height: 570px;
   padding-bottom: 0;
+  border-radius: 24px;
 `
 
 const TokenInfoContainer = styled.div`
   > div {
     margin-bottom: 20px;
   }
-
```

Comment:
don't delete this
![Screenshot 2022-08-05 at 2 43 59 PM](https://user-images.githubusercontent.com/98292246/183017847-bb0e90a6-2c83-4ad9-a900-5b20fe38c327.png)